.github/workflows/reusable-setup.yml

```
@@ -0,0 +1,220 @@
+              echo "CC_armv7_unknown_linux_gnueabihf=arm-linux-gnueabihf-gcc" >> $GITHUB_ENV
+              echo "CXX_armv7_unknown_linux_gnueabihf=arm-linux-gnueabihf-g++" >> $GITHUB_ENV
+              echo "AR_armv7_unknown_linux_gnueabihf=arm-linux-gnueabihf-ar" >> $GITHUB_ENV
+              echo "PKG_CONFIG_PATH=/usr/lib/arm-linux-gnueabihf/pkgconfig" >> $GITHUB_ENV
+              ;;
+          esac
+          
+          case "${{ inputs.install-cross-tools }}" in
+            "musl"|"all")
+              echo "Installing musl cross-compilation tools..."
+              sudo apt-get install -y musl-tools musl-dev
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)